apps/web/.env.example

```diff
@@ -4,7 +4,7 @@ OAUTH_POST_LOGOUT_REDIRECT="http://localhost:8000/logged-out"
 
 # Mock auth endpoint
 OAUTH_LOGOUT_URI="http://localhost:8000/oxauth/endsession"
-# SiC endpoint (will break e2e tests)
+# SiC endpoint ( will break e2e tests )
```

Comment:
Is there a reason for a space after the opening parenthesis and before the closing parenthesis?

Reply:
Yeah, I just needed something to trigger full CI.  The change itself has no meaning.

Reply:
@petertgiles May I suggest another change instead:

`# Logging - enable by setting the logging level.  Eight severity levels from rfc5424.`
to
`# Logging - enable by setting the logging level. Eight severity levels from rfc5424.`